mapie/regression.py

```diff
@@ -322,7 +322,7 @@ def _check_estimator(
             return LinearRegression()
         if not (hasattr(estimator, "fit") and hasattr(estimator, "predict")):
             raise ValueError(
-                "Invalid estimator. "
+                "Invalid estimator."
```

Comment:
Can use the new `check_estimator_fit_predict` function ?

Reply:
done